detector/detector.go

```diff
@@ -40,6 +40,13 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+const (
```

Comment:
this should be dropped, it's not used - I'll clean this up.

Reply:
Are we using these constants or should we remove them?

Reply:
I dropped them, they were not used